src/workflow/workflow.ts

```diff
@@ -1,31 +1,31 @@
-import { IPkName, IRepositoryWritable } from "../repository/repository";
+import { IPkName, IRepositoryWritable } from "../repository/repository"
 
 export interface Context {
```

Comment:
Rename to WorkflowContext

Reply:
fixed by [9dec683](https://github.com/all41-dev/server/pull/37/commits/9dec6838e68d41361af6eb59aa00d23023ba2105)